src/System.Runtime.Extensions/src/System/CodeDom/Compiler/IndentedTextWriter.cs

```diff
@@ -213,28 +212,28 @@ public override void WriteLine(long value)
             _tabsPending = true;
         }
 
-        public override void WriteLine(object value)
+        public override void WriteLine(object? value)
         {
             OutputTabs();
             _writer.WriteLine(value);
             _tabsPending = true;
         }
 
-        public override void WriteLine(string format, object arg0)
+        public override void WriteLine(string format, object? arg0)
         {
             OutputTabs();
             _writer.WriteLine(format, arg0);
             _tabsPending = true;
         }
 
-        public override void WriteLine(string format, object arg0, object arg1)
+        public override void WriteLine(string format, object? arg0, object? arg1)
         {
             OutputTabs();
             _writer.WriteLine(format, arg0, arg1);
             _tabsPending = true;
         }
 
-        public override void WriteLine(string format, params object[] arg)
+        public override void WriteLine(string format, params object?[] arg)
```

Comment:
double checking: should arg (outer) be nullable

Reply:
nope.